src/Markup/Perspex.Markup/Data/Plugins/ExceptionValidationCheckerPlugin.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Perspex.Data;
+
+namespace Perspex.Markup.Data.Plugins
+{
+    /// <summary>
+    /// Validates properties that report errors by throwing exceptions.
+    /// </summary>
+    public class ExceptionValidationCheckerPlugin : IValidationCheckerPlugin
```

Comment:
Do we need "checker" in the names here? Seems kinda redundant. IMO `IValidationPlugin`, `ExceptionValidationPlugin` would suffice.